eng/pipelines/runtime.yml

```diff
@@ -1078,7 +1078,10 @@ jobs:
       liveRuntimeBuildConfig: release
       runtimeVariant: llvmaot
       condition: >-
-        eq(variables['isFullMatrix'], true)
+        or(
+          eq(dependencies.evaluate_paths.outputs['SetPathVars_libraries.containsChange'], true),
+          eq(dependencies.evaluate_paths.outputs['SetPathVars_mono.containsChange'], true),
```

Comment:
We should also run the tests if `SetPathVars_runtimetests.containsChange` is true, I think